hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/client/utils/SparkMetadataWriterUtils.java

```diff
@@ -226,6 +230,21 @@ public static ExpressionIndexComputationMetadata getExpressionIndexRecordsUsingB
     return new ExpressionIndexComputationMetadata(HoodieJavaRDD.of(bloomFilterRecords.javaRDD()));
   }
 
+  private static void setBloomFilterProps(HoodieWriteConfig metadataWriteConfig, HoodieIndexDefinition indexDefinition) {
+    if (indexDefinition.getIndexOptions().containsKey(HoodieExpressionIndex.FALSE_POSITIVE_RATE)) {
```

Comment:
nit: would be better to have a config key mapping of `Map<String, String>` to store `HoodieExpressionIndex.FALSE_POSITIVE_RATE -> HoodieStorageConfig.BLOOM_FILTER_FPP_VALUE.key()` so we can avoid repeated code on setting each property.